File: BedwyrKnx.GUI/Modules/GlobalVariables.vb
'Imports Ouroboros.Authorization.Iris
Imports Ouroboros.AuthManager.Eos

Module GlobalVariables

    ''' <summary>
    ''' 授权信息
    ''' </summary>
    'Public _AuthInfo As AuthorizationInfoCollection
    Public _AuthInfo As AuthInfo

    ''' <summary>
    ''' 程序配置
    ''' </summary>
    Public AppConfig As New AppConfigManager

    ''' <summary>
    ''' KNX系统对象
    ''' </summary>
    Public KS As KnxSystem

End Module

File: BedivereKnx.GUI/GlobalVariables.cs
using Ouroboros.AuthManager.Eos;

namespace BedivereKnx.GUI
{

    internal static class Globals
    {

        /// <summary>
        /// 程序集信息
        /// </summary>
        internal static AssemblyInfo AssemblyInfo = new();

        /// <summary>
        /// 程序配置
        /// </summary>
        internal static AppConfigManager AppConfig = new();

        /// <summary>
        /// 授权信息
        /// </summary>
        internal static AuthInfo? AuthInfo;

        /// <summary>
        /// KNX系统对象
        /// </summary>
        internal static KnxSystem? KnxSys;

    }

}
